modules/claims_api/lib/bgs_service/benefit_claim_service.rb

```diff
@@ -17,5 +17,17 @@ def find_bnft_claim(claim_id:)
 
       make_request(endpoint: bean_name, action: 'findBnftClaim', body:)
     end
+
+    def update_bnft_claim(claim_id:)
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <bnftClaimId />
+      EOXML
+
+      { bnftClaimId: claim_id }.each do |k, v|
+        body.xpath("./*[local-name()='#{k}']")[0].content = v
+      end
+
+      make_request(endpoint: bean_name, action: 'updateBnftClaim', body:)
+    end
```

Comment:
so I think this method is what's missing from the other PR, hence why I commented it doesn't exist, but it does, just in the future..... 😆